README.md

```diff
@@ -1,14 +1,17 @@
-# Weave GitOps Toolkit
+# Weave libgitops
 
 A set of packages to help build Git-backed applications.
-Weave GitOps Toolkit builds on top of the [Kubernetes API Machinery](https://github.com/kubernetes/apimachinery).
+Weave `libgitops` builds on top of the [Kubernetes API Machinery](https://github.com/kubernetes/apimachinery).
 
 ## Getting Help
 
-If you have any questions about, feedback for or problems with `gitops-toolkit`:
+If you have any questions about, feedback for or problems with `libgitops`:
 
-- Invite yourself to the <a href="https://slack.weave.works/" target="_blank">Weave Users Slack</a>.
-- Ask a question on the [#ignite](https://weave-community.slack.com/messages/general/) slack channel.
-- [File an issue](https://github.com/weaveworks/gitops-toolkit/issues/new).
+- Invite yourself to the [Weave Users Slack](https://slack.weave.works/).
+- Ask a question on the [#ignite](https://weave-community.slack.com/messages/general/) Slack channel.
+- [File an issue](https://github.com/weaveworks/libgitops/issues/new).
 
 Your feedback is always welcome!
+
+## Notes
+This project was formerly called `gitops-toolkit`, but has now been given a more descriptive name.
```

Comment:
Also add a comment and link to the "real" GitOps Toolkit at https://github.com/fluxcd/toolkit

Reply:
:+1: